ui/src/app/components/runtime-configuration-panel.rtl.spec.tsx

```diff
@@ -2453,110 +2252,4 @@ describe(RuntimeConfigurationPanel.name, () => {
       spinDiskElement('num-preemptible');
     expect(preemptibleCountInput.disabled).toBeTruthy();
   });
-
-  // These tests require a little more simplification, so we are skipping them for now. However, they will run as
-  // part of runtime-configuration-panel.enzyme.spec.tsx.
-  test.skip.each([
```

Comment:
need equivalent tests for:
* disk type change -> delete disk and runtime, re-create both
* enable GPU -> delete and recreate runtime but not disk

Reply:
Replaced with "should delete and recreate both the disk and runtime when the disk type is changed"